manager/build.sh

```diff
@@ -1,10 +1,17 @@
 #!/bin/bash
 
-#Need to login first
-OPENSHIFT_TOKEN=$(oc whoami -t)
-docker login -u system -p ${OPENSHIFT_TOKEN} 172.30.1.1:5000
- 
+echo '*********************************'
 echo 'Building CloudEvent Flow Manager'
-docker build -t cef-manager .
-docker tag cef-manager:latest 172.30.1.1:5000/myproject/cef-manager:latest
-docker push 172.30.1.1:5000/myproject/cef-manager:latest
+echo '*********************************'
+mvn -f ../model/pom.xml clean install ||  exit 1;
```

Comment:
I think, instead of this script, we should use the `dmp` (`docker-maven-plugin`), like here:

* https://github.com/aerogear/aerogear-unifiedpush-server/blob/master/servers/plain/pom.xml#L116-L148
* https://github.com/aerogear/aerogear-unifiedpush-server/tree/master/servers/plain/src/main/docker